velox/dwio/dwrf/reader/SelectiveTimestampColumnReader.cpp

```diff
@@ -72,6 +72,13 @@ void SelectiveTimestampColumnReader::read(
   VELOX_CHECK(
       !scanSpec_->valueHook(),
       "Selective reader for TIMESTAMP doesn't support aggregation pushdown yet");
+  if (!resultNulls_) {
```

Comment:
You also need to reallocate it if it is not large enough

Reply:
Could you elaborate this a bit?